VersionMiner/Services/IAppService.cs

```diff
@@ -0,0 +1,28 @@
+﻿// <copyright file="IAppService.cs" company="KinsonDigital">
+// Copyright (c) KinsonDigital. All rights reserved.
+// </copyright>
+
+namespace VersionMiner.Services;
+
+/// <summary>
+/// Provides application type functionality.
+/// </summary>
+public interface IAppService
+{
+    /// <summary>
+    /// Exits the application with the given exit <paramref name="code"/>.
+    /// </summary>
+    /// <param name="code">The exit code to use.</param>
+    void Exit(int code);
+
+    /// <summary>
+    /// Exits the application with the no error code of 0.
```

Comment:
Remove "the" and replace with "a"
```suggestion
    /// Exits the application with a no error code of 0.
```

Reply:
I refactored to this instead.  Let me know if this is ok.

```suggestion
    /// Exits the application with the no error code.
```